mandatory/thinking-like-a-programmer/task.md

```diff
@@ -31,10 +31,15 @@ After you've watched these videos I'd like you to answer these questions
 
 <!-- Write your answer here -->
 
+ A great programmer is able to understand problems clearly, break them down into hypotheses, and propose solutions in a coherent manner. They understand concepts quickly, or ask the right questions to help make them clear, and don’t need to have everything written down in a specifications document.
```

Comment:
yes asking more questions in order to understand the problem better is a really important thing to do!